server/rpc/rpcserverhelp.go

```diff
@@ -223,28 +223,13 @@ var helpDescsEnUS = map[string]string{
 	"txRawResult-vin":           "The transaction inputs as JSON objects",
 	"txRawResult-vout":          "The transaction outputs as JSON objects",
 	"txRawResult-blockHash":     "Hash of the block the transaction is part of",
-	"txRawResult-confirmations": "Number of confirmations of the block (Will be 'null' if txindex is not disabled)",
+	"txRawResult-confirmations": "Number of confirmations of the block",
```

Comment:
It looks like you deleted this everywhere, shouldn't it be delete there as well?
I did ask you to maybe restore it at least in part of the places, so this comment is only relevant in case we decide that restoring is not an option.